applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXAdViewFactory.java

```diff
@@ -33,6 +33,24 @@ public AppLovinMAXAdViewFactory(final BinaryMessenger messenger)
     @Override
     public PlatformView create(@Nullable final Context context, final int viewId, final Object args)
     {
+        if ( !AppLovinMAX.getInstance().isInitialized() )
```

Comment:
I don't want to leave anything unchecked, can you lmk what your reason for this is? I'd expect publishers to mount the widget as soon as the app launches.

Reply:
Right, I'm confused about being initialized.  Will update.